src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -36,3 +36,10 @@ export type Event = {
   message: string;
   platform?: string;
 };
+
+export type EventsStatsData = [number, {count: number}[]][];
+
+export type EventsStats = {
+  data: EventsStatsData;
+  totals?: any;
```

Comment:
We should type this properly. Looking at https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/snuba.py#L327 it seems like it should be something like`{'count': number}`